src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADDING_BOOK = 'books/ADDING_BOOK';
+const REMOVING_BOOK = 'books/REMOVING_BOOK';
```

Comment:
- [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions like:
```javascript
<app name>/<reducer name>/<action type>
```
for example:
```javascript
const REMOVE_BOOK = 'react-bookstore/book/REMOVE_BOOK';
```

Reply:
@gemmen29 thank you1